ydb/docs/en/core/reference/ydb-ports.md

```diff
@@ -0,0 +1,26 @@
+# Default ports assigned to {{ ydb-short-name }}
+
+In the instructions for working with {{ ydb-short-name }}, the following main ports assigned by default are used:
+
+| Port  | Purpose                                             |
+|-------|-----------------------------------------------------|
+| 2135  | Port for secure gRPC connections (TLS).             |
+| 2136  | Port for gRPC connections.                          |
+| 19001 | Interconnect port.                                  |
+| 8765  | HTTPS port for cluster monitoring.                  |
+| 4317  | Port for query tracing logs.                        |
+
+Secondary ports configured by default for integrations and external tools:
+
+| Port  | Purpose                                                                 |
+|-------|-------------------------------------------------------------------------|
+| 2130  | Port for connecting the connector.                                      |
+| 5432  | Port for connecting to PostgreSQL.                                      |
```

Comment:
add a comment saying it's for both connecting to PostgreSQL in federated queries and for YDB's PostgreSQL compatibility mode